config/initializers/rescue_from.rb

```diff
@@ -1,16 +1,54 @@
 require 'openstax_rescue_from'
 
-secrets = Rails.application.secrets
-exception_secrets = secrets.exception
 OpenStax::RescueFrom.configure do |config|
-  # Show the default Rails exception debugging page on dev
-  config.raise_exceptions = EnvUtilities.load_boolean(name: 'RAISE',
-                                                      default: Rails.env.development?)
+  config.raise_exceptions = [false, 'false'].exclude?(ENV.fetch('RAISE_EXCEPTIONS', nil)) ||
+                            Rails.application.config.consider_all_requests_local
+  config.raise_background_exceptions = [false, 'false'].exclude?(ENV.fetch('RAISE_BACKGROUND_EXCEPTIONS', nil))
 
-  config.app_name = 'Accounts'
-  config.contact_name = exception_secrets[:contact_name]&.html_safe
+  config.app_name = ENV.fetch('APP_NAME', nil)
+  config.contact_name = ENV.fetch('EXCEPTION_CONTACT_NAME', nil)
```

Comment:
I suggest you load all the ENV vars in secrets.yml when possible so you don't go crazy later trying to find all spots that load some variable. Not possible for config/puma.rb I think, but the rest shoudl be fine?